tests/integration/llm/client.py

```diff
@@ -1165,4 +1165,4 @@ def run(raw_args):
 
 
 if __name__ == "__main__":
-    run(sys.argv)
+    run(sys.argv[1:])
```

Comment:
does this work with

```
parser.parse_args(args=raw_args)
```
?

Reply:
That is what it is fixing. The sys.argv includes the executable `client.py` which is being removed here. Passing it to `parse_args` will have it treat the `client.py` as the first argument